app/templates/components/add-cron-job.hbs

```diff
@@ -1,32 +1,70 @@
-<form {{action (perform save) on="submit"}}>
-  <div class="form-elem select">
-    <span class="label">Branch</span>
-    {{#x-select value=selectedBranch required="true" on-change=(action (mut selectedBranch)) as |xs|}}
-      {{#each branches as |branch|}}
-        {{#xs.option value=branch}}{{branch.name}}{{/xs.option}}
-      {{/each}}
-    {{/x-select}}
+{{#travis-form 
+  class='form--cron' 
+  onSubmit=(action 'handleSubmit') as |form| }}
+  <div class='form-elem select'>
+    {{#form.field
+      class='branch-selector'
+      label='Branch'
+      requiresValidation=false
+      required=true
+      value=selectedBranch
+      onChange=(action (mut selectedBranch))
+      as |field|
+    }}
+      {{#field.select 
+        searchField='name' 
+        placeholder='Type branch name'
+        oninput=(action 'onQueryChange') 
+        options=comparedBranches as |branch|}}
+        {{branch.name}}
+      {{else}}
+        <div class='ember-power-select-option ember-power-select-option--no-matches-message'>
+          {{#if branchesFetcher.loadingData}}
+            Loading...
+            {{else}}
+            No results found
+          {{/if}}
+        </div>
+      {{/field.select}}
+    {{/form.field}}
   </div>
-  <div class="form-elem select">
-    <span class="label">Interval</span>
-    {{#x-select value=selectedInterval required="true" on-change=(action (mut selectedInterval)) as |xs|}}
-      {{#each intervals as |interval|}}
-        {{#xs.option value=interval}}{{interval}}{{/xs.option}}
-      {{/each}}
-    {{/x-select}}
+  <div class='form-elem select'>
+    {{#form.field
+      label='Interval'
+      value=selectedInterval
+      requiresValidation=false
+      required=true
+      onChange=(action (mut selectedInterval))
+      as |field|
+    }}
+      {{#field.select options=intervals as |interval|}}
+        {{interval}}
+      {{/field.select}}
+    {{/form.field}}
   </div>
-  <div class="form-elem select">
-    <span class="label">Options</span>
-    {{#x-select value=selectedOption required="false" on-change=(action (mut selectedOption)) as |xs|}}
-      {{#xs.option value=false}}{{options.[0]}}{{/xs.option}}
-      {{#xs.option value=true}}{{options.[1]}}{{/xs.option}}
-    {{/x-select}}
+  <div class='form-elem select'>
+    {{#form.field
+      label='Options'
+      value=selectedOption 
+      requiresValidation=false
```

Comment:
There's no need to set this property externally, it is a computed property that derives its value depending on the component settings. This may cause unexpected results, because the component is designed differently. Please avoid overriding computed properties of components as in most cases it means misuse of the component.

Reply:
This is for hide error messges

Reply:
Ah, I see, that makes sense! Thanks for the clarification 👍